### PR TITLE
Revive clobbered translations 2

### LINKED ISSUE #56: Warning: commit #23 has rolled back some translations
I saw that commit #23 (7099d874775890c9811a37861e6075f9a2e084ac) has changed some translations into their older states. This is the case for the French translation, where my contributions have been lost in the process, and some other translations may have been impacted. Just to warn you!

### PR DESCRIPTION
**Pull request by ann-kilzer**
Attempt to revive the clobbered translations. 🙇🏻‍♀️ 

Resolves #56 

This is delicate because I can't understand all the other languages. However, I am prioritizing files that have been edited and will carefully link prior PRs to check this work. 

Manually checked all the PRs prior to the machine translation commit in #23 and ensured we've kept those authors' changes.

### CODE REVIEW

File: locales/ko.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
 {
```
Comment (ann-kilzer) [inline locales/ko.json:1]: - [ ] Check https://github.com/ourjapanlife/findadoc-localization/commit/a121806eed78863e837e3b855889145e88023228

File: locales/ne.json [hunk 2]
```diff
@@ -1,36 +1,36 @@
 {
```
Comment (ann-kilzer) [inline locales/ne.json:1]: - [ ] Check https://github.com/ourjapanlife/findadoc-localization/commit/a6412defadeccaaee283d5678a5cd4de08f68a90

File: locales/fil.json [hunk 3]
```diff
@@ -1,29 +1,29 @@
 {
```
Comment (ann-kilzer) [inline locales/fil.json:1]: Check: https://github.com/ourjapanlife/findadoc-localization/commit/00779b8cff07e4859899a20ebee8370a7749fb81

File: locales/es.json [hunk 4]
```diff
@@ -42,7 +42,6 @@
         "header": {
```
Comment (ann-kilzer) [inline locales/es.json:42]: - [ ] Check https://github.com/ourjapanlife/findadoc-localization/commit/f9b1baa2ff7450f07951fa613b74c8182fdd131c

File: locales/id.json [hunk 5]
```diff
@@ -2,34 +2,34 @@
     "about": {
```
Comment (ann-kilzer) [inline locales/id.json:2]: - [ ] check: https://github.com/ourjapanlife/findadoc-localization/commit/d0270a0a9887d1733c79d1d10d647dbf78be3248

File: locales/es.json (hunk 4 above)
Comment (ann-kilzer) [reply locales/es.json:42]: Since this file was first made outside lokalize, it wasn’t clobbered

File: locales/fil.json (hunk 3 above)
Comment (ann-kilzer) [reply locales/fil.json:1]: confirmed updates here: https://github.com/ourjapanlife/findadoc-localization/commit/87c42fd0f01ec3eeddcd3f54456624e8465cc6cc

File: locales/ms.json [hunk 6]
```diff
@@ -40,24 +40,23 @@
     },
```
Comment (ann-kilzer) [inline locales/ms.json:40]: - [ ] Check: https://github.com/ourjapanlife/findadoc-localization/commit/863f1f5e13650de8f357c7038c72d7feda1e5a68

File: locales/vi.json [hunk 7]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (ann-kilzer) [inline locales/vi.json:1]: - [ ] check https://github.com/ourjapanlife/findadoc-localization/commit/c4264c67b606ddc2ce0a7470cb28bdfa65c83540

**Comment by ann-kilzer** [comment]
@richoux Let me know your thoughts!

File: locales/id.json [hunk 8]
```diff
@@ -80,8 +79,8 @@
         "delete": "Delete"
     },
     "index": {
-        "massSite": "Siapa pun yang berusia 18 tahun ke atas yang memiliki voucher sekarang dapat divaksinasi di situs vaksinasi massal Tokyo atau Osaka.",
-        "register": "Pergi ke Situs Pendaftaran",
+        "massSite": "Anyone 18 years or older that has a voucher can now get vaccinated at the Tokyo or Osaka mass vaccination sites.",
```
Comment (Ermish) [inline locales/id.json:82]: not sure if this one is right

File: locales/ko.json [hunk 9]
```diff
@@ -80,12 +79,12 @@
         "delete": "Delete"
     },
     "index": {
-        "massSite": "바우처가있는 18 세 이상이면 누구나 도쿄 또는 오사카 대량 예방 접종 장소에서 예방 접종을받을 수 있습니다.",
-        "register": "등록 사이트로 이동",
-        "title": "일본의 COVID-19 예방 접종 취소 대기자 명단"
+        "massSite": "Anyone 18 years or older that has a voucher can now get vaccinated at the Tokyo or Osaka mass vaccination sites.",
```
Comment (Ermish) [inline locales/ko.json:82]: not sure if this one is right

File: locales/ne.json [hunk 10]
```diff
@@ -80,14 +79,14 @@
         "delete": "Delete"
     },
     "index": {
-        "massSite": "१ 18 बर्ष वा माथिका जोसँग भाउचर छ अब टोकियो वा ओसाका मास खोप साइटहरूमा खोप लिन सकिन्छ।",
-        "register": "पञ्जीकरण साइटमा जानुहोस्",
-        "title": "COVID-19 जापानमा खोप रद्द गर्ने प्रतीक्षा सूचीहरू"
+        "massSite": "Anyone 18 years or older that has a voucher can now get vaccinated at the Tokyo or Osaka mass vaccination sites.",
```
Comment (Ermish) [inline locales/ne.json:82]: same

**Review by Ermish — APPROVED** [review]
LGTM if that one section is correct.

**Review by theyokohamalife — APPROVED** [review]
Thank you!

**Comment by richoux** [comment]
@ann-kilzer I don't have an opinion about other languages, I can just tell you that the French version need some modifications. I can handle them once this PR is accepted.

File: locales/id.json (hunk 8 above)
Comment (ann-kilzer) [reply locales/id.json:82]: Good catch! Will fix tonight

**Comment by ann-kilzer** [comment]
@richoux many thanks for your help! that sounds good

File: locales/ko.json (hunk 9 above)
Comment (ann-kilzer) [reply locales/ko.json]: fixed

**Pull request merged**